Training/python/scaling_utils.py

```diff
@@ -216,6 +216,7 @@ def get_quantiles(var_array):
     """
     quantile_dict = {}
     var_array = ak.to_numpy(ak.flatten(var_array, axis=-1))
+    if np.size(var_array)==0: return {'median': 0., 'min': 0., 'max': 0., '1sigma': {'left': 0., 'right': 0.}, '2sigma': {'left': 0., 'right': 0.}, '3sigma': {'left': 0., 'right': 0.}, '5sigma': {'left': 0., 'right': 0.}}
```

Comment:
The same comment here as for `compute_mean()/compute_std()` functions, maybe it's better to return `None` too?

Reply:
Ok, I changed this, as well as in the other parts, but having `None` values in the quantiles breaks plot_quantile_ranges.py, so I added a special treatment there too